### ISSUE TITLE
Running STFService.apk

### ISSUE THREAD

**Issue by Creatik** [openstf/minitouch#66]
Good afternoon, but could you tell us about STFService.apk. A mini-instruction that needs to be executed so that the minitouch redirects to STFService.apk. Need to start a service and agent?

Thanks a lot in advance

**Comment by koral--** [comment]
Do you mean this: https://github.com/openstf/STFService.apk#running-the-service ?

**Comment by Creatik** [comment]
Yes. I am using your Minitouch in an AAMirror project. and currently does not work on Android 10. How to redirect correctly.

**Comment by Creatik** [comment]
Here is the minitouch connection filehttps://github.com/slashmax/AAMirror/blob/master/mirror/src/main/java/com/github/slashmax/aamirror/MinitouchSocket.java
